app/views/robot_verifications/submission.html.erb

```diff
@@ -16,55 +16,72 @@
     </tr>
   </thead>
   <tbody>
-  <% if @all_labels.last.nil? -%>
```

Comment:
More explicit. Especially important considering this was one cause of the buggy behaviour